### PR TITLE
refactor(@vtmn/css): add line-heights to buttons

### PR DESCRIPTION
**Pull request by crotru**
## Pull Request checklist

🚨 Please review the [guidelines for contributing](../blob/main/CONTRIBUTING.md) to this repository.

- [x] Make sure you are requesting to **pull a topic/feature/bugfix branch**. Don't request your main!
- [x] Make sure you are making a pull request against the **main branch** (left side).
- [x] Check the commit's or even all commits' message styles matches our requested structure.
- [x] Check your code additions will fail neither code linting checks nor unit test.

## Description

Add line-heights to cta in order to give them a fitting default height.

According to [design-system statements](https://www.decathlon.design/726f8c765/p/8008f8-button/t/58ed03) :

**S should be 32px** 
that's 2rem 
that's **_0.875rem_** (font-size) * **_~1.2857_** (line-height) + **_0.4375rem_** (padding-top) + **_0.4375rem_** (padding-bottom)

**M should be 48px** 
that's 3rem 
that's **_1rem_** (font-size) * **_1.25_** (line-height) + **_0.875rem_** (padding-top) + **_0.875rem_** (padding-bottom)

**L should be 64px** 
that's 4rem 
that's **_1.25rem_** (font-size) * **_1.2_** (line-height) + **_1.25rem_** (padding-top) + **_1.25rem_** (padding-bottom)

### CODE REVIEW

File: packages/sources/css/src/components/button/src/index.css [hunk 1]
```diff
@@ -164,19 +165,22 @@
 
   /* Sizes */
   .vtmn-btn_size--small {
-    font-size: 0.75rem;
-    padding: 0.5rem 1rem;
+    font-size: 0.875rem;
+    padding: 0.4375rem 1rem;
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:169]: Why this padding? 
I've followed https://www.decathlon.design/726f8c765/p/8008f8-button/b/50afe1

image

**Review by lauthieb — COMMENTED** [review]
Thanks Charles!

File: packages/sources/css/src/components/button/src/index.css [hunk 2]
```diff
@@ -164,19 +165,22 @@
...
+    line-height: calc(1.125 / 0.875);
```
Comment (lauthieb) [inline packages/sources/css/src/components/button/src/index.css:170]: Why you've setted a `calc` for the small class and not for the others (medium & large)?

**Review by lauthieb — APPROVED** [review]

**Comment by lauthieb** [comment]
@crotru FYI, we don't have a WIP plugin. Instead, you can use the draft PR built-in feature in GitHub.

**Review by RomainJoly — APPROVED** [review]

**Pull request merged**